### DISCUSSION TITLE
Arch ROCm installation path

### DISCUSSION THREAD

**Discussion by sofiageo** [rocm-arch/rocm-arch#979] [Ideas]
Arch ROCm should either follow the ROCm [filesystem hierarchy](https://rocm.docs.amd.com/en/docs-6.0.0/conceptual/file-reorg.html) or the Arch Linux [filesystem hierarchy](https://man.archlinux.org/man/file-hierarchy.7). It currently follows neither. 

For example this is how it looks for debian: https://packages.debian.org/sid/amd64/libhipsparse-dev/filelist

**Comment by tpkessler** [comment]
Some third party libraries like pytorch or tensorflow still use pre-FHS include or library paths. It takes them some time to follow the new FHS. Transition to FHS should nevertheless be almost complete with ROCm 6. It's currently in `staging` and will be in `testing` soon.

**Comment by sofiageo** [comment]
Thanks @tpkessler - just to be clear, I also mean that /opt/rocm-ver should include the version number of the release. 

p.s The only reason I made this issue is because users are complaining to me that my package `opencl-amd` should not include a soft link of /opt/rocm to /opt/rocm-6.0.0. IMO if other libraries like magma-hip are not installed in /opt/rocm, it should not matter.
